cmd/cli/bulk.go

```diff
@@ -46,45 +47,57 @@ type BulkResponseItemWork struct {
 
 // queryBulk queries a bulk set of servers using a query file.
 func queryBulk(file string) error {
-	f, err := os.Open(file)
-	if err != nil {
-		log.Fatal(err)
+	// To simplify the workerpool load all the entries we are going to work on
+	lines := fileLines(file)
+
+	if len(lines) > 10000 {
+		return fmt.Errorf("too many servers requested %d (max 10000)", len(lines))
```

Comment:
Two aspects for this, channels have a maximum size, which while large isn't infinite and if this gets too big i'd have to have a more complex runner which i don't think is needed. I think it's also reasonable to be limit the request to the number of servers that a call like this could process in a few minutes

Reply:
Sure channels have limits, but it doesn't need to be able to consume the whole dataset into the buffer, allowing producers to block while workers consume the requests is fine.

My question would be is it up to the consumer to decided if they want to wait a long time?

Reply:
This shouldn't block this being merged.

Reply:
ty, i'll get this in for now so we can validate the query part and then look at getting that in